packaging/installer/README.md

```diff
@@ -165,3 +165,63 @@ the community helps fix any bugs that might have been introduced in previous rel
 -   Protect yourself from the rare instance when major bugs slip through our testing and negatively affect a Netdata
     installation
 -   Retain more control over the Netdata version you use
+
+## Installation notes and known issues
+
+We are tracking a few issues related to installation and packaging.
+
+### Failure to claim/connect to Netdata Cloud
+
+If you're running an older Linux distribution or one that has reached EOL, such as Ubuntu 14.04 LTS, Debian 8, or CentOS
+6/7, your Agent may not be able to connect to Cloud due to an old version of OpenSSL that is incompatible with Netdata
+Cloud.
+
+We recommend using the [static build installation](/packaging/installer/methods/kickstart-64.md) to avoid this issue.
+You can, however, use the workaround below.
+
+#### OpenSSL workaround
+
+Instead of using the static build, you can also reconfigure OpenSSL to allow self-signed certificates and skip hostname
+checks.
+
+> ⚠️ Using this workaround disables security checks in OpenSSL and reduces the security of your system and its metrics.
+> Allowing self-signed certificates and skipping hostname checks could allow a hostile intermediary to collect your
+> node's metrics as it transits to Netdata Cloud.
+
+If you choose to proceed, you must export the `-DACLK_SSL_ALLOW_SELF_SIGNED` CFLAG before installation.
+
+```bash
+export CFLAGS="-DACLK_SSL_ALLOW_SELF_SIGNED"
+```
+
+Then install Netdata using your method of choice.
+
+#### Ubuntu 14.04 LTS
+
+To use the `CFLAGS` workaround on Ubuntu 14.04 LTS, you must first install `libuv1-dev` via a PPA:
+
+```bash
+add-apt-repository ppa:acooks/libwebsockets6
+apt-get update
+apt-get install libuv1-dev
+```
+
+Then proceed with the CFLAGS workaround described above.
+
+### CentOS 6 and CentOS 8
+
+To install the Agent on certain CentOS and RHEL systems, you must enable non-default repositories, such as EPEL or
+PowerTools, to gather hard dependencies. See the [CentOS 6](/packaging/installer/methods/manual.md#centos-rehel-6-x) and
+[CentOS 8](/packaging/installer/methods/manual.md#centos-rehel-8-x) sections for more information.
```

Comment:
Why is this here? The installer takes care of this, it even tells the user and prompts then (_optionally_).

Reply:
This is here because of the linked sections ([6](https://learn.netdata.cloud/docs/agent/packaging/installer/methods/manual/#centos--rhel-6x) and [8](https://learn.netdata.cloud/docs/agent/packaging/installer/methods/manual/#centos--rhel-6x)), which were added some months ago to explain how to install on these distros, since users were running into issues.

If those sections are no longer needed, let me know and I'll remove them in a separate PR.

Reply:
I would just add a note to say that our installer automatically takes cares of this if you use our `kickstart.sh`.